recipes/flang-rt/meta.yaml

```diff
@@ -0,0 +1,111 @@
+{% set version = "21.0.0.dev1" %}
+{% set major = version.split(".")[0] %}
+
+package:
+  name: flang-rt-split
+  version: {{ version }}
+
+source:
+  # url: https://github.com/llvm/llvm-project/archive/refs/tags/llvmorg-{{ version.replace(".rc", "-rc") }}.tar.gz
+  url: https://github.com/llvm/llvm-project/archive/e5129b7e2092c54e503f60cc0ac552c765d5a144.tar.gz
+  sha256: f8506cf5d53a5d555384a8917432db40ef8a2b89a5172d64f8a753b649f0274b
+  patches:
+    - patches/0001-avoid-baking-in-build-time-paths-to-compiler-rt-into.patch
+
+build:
+  number: 0
+  # intentionally only windows (main target) & linux (debuggability)
+  skip: true  # [osx]
+
+requirements:
+  build:
+    - cmake
+    - ninja
+    - {{ stdlib('c') }}
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - flang {{ version }}.*
+  host:
+    - clangdev =={{ version }}
+    - compiler-rt =={{ version }}
+    - llvmdev =={{ version }}
+    - mlir =={{ version }}
+
+outputs:
+  - name: libflang-rt
+    build:
+      # not supported on windows yet, see
+      # https://github.com/llvm/llvm-project/issues/134186
+      skip: true  # [win]
+    files:
+      - lib/libflang_rt.runtime.so                                                          # [linux]
+      - lib/clang/{{ major }}/lib/x86_64-unknown-linux-gnu/libflang_rt.runtime.so           # [linux]
```

Comment:
This should be in the target specific output.

Reply:
I don't understand. This is the actual location for the shared library that's backing the `lib/libflang_rt.runtime.so` symlink. It needs to be in the `libflang-rt` output, unless you want to _also_ introduce `libflang-rt_{{ target_platform }}`, which is not what I understood from
> A shared library should be `libflang-rt` and static libraries should be `flang-rt_{{ target_platform }}`.

Reply:
`lib/clang/{{ major }}/lib/x86_64-unknown-linux-gnu/libflang_rt.runtime.so` should be in `flang-rt_{{ target_platform }}`, and `lib/libflang_rt.runtime.so` should be a copy of it.

This is analogous to having both `x86_64-conda-linux-gnu/lib/libstdc++.so`  and `lib/libstdc++.so`

Reply:
ok, done.

Reply:
Thanks